### PR TITLE
Add support for GTM

### PR DESCRIPTION
**Pull request by varmar05**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "export": "vuepress export src"
   },
   "devDependencies": {
-    "@vuepress/plugin-google-analytics": "^1.8.2",
+    "vuepress-plugin-google-tag-manager": "*",
```
Comment (mostlyAtNight) [inline package.json:18]: Should we be explicit with the version here?

File: src/.vuepress/components/AppDownload.vue [hunk 2]
```diff
@@ -12,7 +12,21 @@
...
 export default {
   name: "appdownload",
-  props: {}
+  props: {},
+  methods: {
+    gtmEvent: function (store) {
+      if (!this.$gtm) return
+
+      this.$gtm.trackEvent({
+        event: null, // Event type [default = 'interaction'] (Optional)
```
Comment (mostlyAtNight) [inline src/.vuepress/components/AppDownload.vue:21]: Are these resulting values all in-line with the conventions we use at present in GA and friends?

File: package.json (hunk 1 above)
Comment (varmar05) [reply package.json:18]: yea, I can put it here, however it is bit experimental plugin with version 0.0.3 or something like that

File: src/.vuepress/components/AppDownload.vue (hunk 2 above)
Comment (varmar05) [reply src/.vuepress/components/AppDownload.vue:21]: It matches GTM docs and I tried to follow conventions from other websites (label, action, category).

But value does not make sense to me, I will remove it.

**Pull request merged**